act-rules-format.md

```diff
@@ -206,6 +216,66 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
   </ul></blockquote>
 </aside>
 
+### Mapping Associated/Affiliated/Secondary Requirements ###
+When an ACT Rule's passed or failed outcomes have different meanings for an accessibility requirements from what is  described in Conformance requirements, these accessibility requirements are mapped as secondary requirements.  
+
+Map an accessibility requirements as secondary for the following rule outcomes: 
+- "AAA": the accessibility requirement is <dfn>not satisfied</dfn> when one or more outcomes for a test target is `failed`, and is <dfn>not satisfied</dfn>when all outcomes are `passed`.
+- "OR": <dfn>further testing is needed</dfn> when one or more of the outcomes for a test target is `failed`, and the accessibility requirement is <dfn>satisfied</dfn> when all outcomes are `passed`,
```

Comment:
And I don't quite understand this name either.

Reply:
"OR" is another temporary label to help identify the purpose of these outcomes. This label will be removed/replaced as we get closer to final.